### DISCUSSION TITLE
Problem with request to https://spatialreference.org/ref/epsg/32719/esriwkt

### DISCUSSION THREAD

**Discussion by AntonioTorga** [OSGeo/grass#3693] [General]
Hi! i'm running a program that should be running great and i got the following message:
No internet connection or epsg code 32719 not found at https://spatialreference.org/ref/epsg/32719/esriwkt

Upon further investigation i realized that https://spatialreference.org/ref/epsg/32719/esriwkt doesn't exist. The valid one is  the same URL but with a .txt file extension at the end.
I don't know if this changed recently or if i really do have an internet connection issue.
If anyone knows what the problem might be i would really appreciate some help.

**Comment by petrasovaa** [comment]
Which program? This doesn't seem to be related to GRASS GIS...

**Reply by AntonioTorga** [reply]
I know think it might be related to FloPy instead of GRASS, sorry

**Comment by neteler** [comment]
@jjimenezshaw may we transfer this to https://github.com/OSGeo/spatialreference.org ?

**Reply by AntonioTorga** [reply]
Yes! thanks

**Comment by jjimenezshaw** [comment]
> @jjimenezshaw may we transfer this to https://github.com/OSGeo/spatialreference.org ?

The site was completely refactored in January. Now it's content is completely static (and much faster).
The url of every srs is kept, but not every format. In your case your are right. The valid url is https://spatialreference.org/ref/epsg/32719/esriwkt.txt

**Reply by AntonioTorga** [reply]
Oh!! thank you, that clears it up

**Comment by jjimenezshaw** [comment]
@AntonioTorga I that enough to close the issue?

**Reply by AntonioTorga** [reply]
yes! thanks
